src/test/java/core/basesyntax/service/impl/CsvParserTest.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.exceptions.NegativeQuantityException;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Operation;
+import core.basesyntax.model.TransactionDto;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class CsvParserTest {
+    private static CsvParser csvParser;
+    private static final String TEST_STRING_1 = "b,banana,20";
+    private static final String TEST_STRING_2 = "s,apple,100";
+    private static final String TEST_STRING_3 = "b,apple,-100";
+    private static List<TransactionDto> expectedDtoList;
+    private static List<String> inputOkStringList;
+    private static List<String> inputNotOkStringList;
+    private static final TransactionDto testDto1 = new TransactionDto(Operation.fromString("b"),
+            new Fruit("banana"), 20);
+    private static final TransactionDto testDto2 = new TransactionDto(Operation.fromString("s"),
```

Comment:
constant naming

or shall we remove these fields?

Reply:
done